README.md

```diff
@@ -24,31 +24,68 @@ Create a new temporary project:
 * With a dependency that have a fixed version:
     `$ cargo-temp anyhow==1.0.13`
 
+### Repositories
+
+You can add repositories to you Cargo.toml.
+
+Examples:
+
+* HTTP
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git`
+
+* SSH
+    `$ cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git`
+
+This will add the repository on the default branch by default. You can choose
+another branch or a revision:
+
+* Branch
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=main`
+
+* Revision
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#rev=7e0f77a38`
+
 ## Features
 
-If you change your mind and decide to keep the project you can just delete the `TO_DELETE` file and the directory
-will not be deleted when the shell exits.
+If you change your mind and decide to keep the project you can just delete the
+`TO_DELETE` file and the directory will not be deleted when the shell exits.
 
 ## Settings
 
 The config file is located at `{CONFIG_DIR}/cargo-temp/config.toml`.
-When you run `cargo-temp` for the first time it will be created automatically
+When you run `cargo-temp` for the first time it will be created automatically.
+We use the [XDG system](https://docs.rs/xdg/2.2.0/xdg/) for both Linux and OSX
+and the [Know Folder system](https://docs.rs/dirs-2/3.0.1/dirs_2/) on Windows.
+
+### `temporary_project_dir`
+
+The path where the temporary projects are created.
+Set on the cache directory by default.
+
+`temporary_project_dir = "/home/name/.cache/cargo-temp/"`
+
+### `cargo_target_dir`
+
+Cargo's target directory override.
+This setting is unset by default and will be ignored if the `CARGO_TARGET_DIR`
+environment variable is already set.
+
+`temporary_project_dir = "home/name/repos/tmp"`
 
-* `temporary_project_dir`: path where the temporary projects are created (cache directory by default).
-* `cargo_target_dir`: cargo's target directory override (unset by default).
-   This setting is ignored if `CARGO_TARGET_DIR` is already set.
-* `editor` and `editor_args`: path to an IDE to start instead of a shell and arguments for it (unset by default).
+### editor (unset by default)
 
-### Example config to run VS Code on Unix
+You can use `editor` and `editor_args` as respectively the path to an IDE to
+start instead of a shell and arguments for it (unset by default).
```

Comment:
It's probably best to rewrite it and try to avoid thinking French and translating from French:

```suggestion
You can use `editor` to start an IDE instead of a shell
and `editor_args` to provide its arguments (unset by default).
```
:point_up: this would avoid the complicated sentences we are so used to use in French XD https://redwoodink.com/resources/avoid-the-word-respectively-for-clear-writing